command/rootchain/initcontracts/init_contracts.go

```diff
@@ -271,44 +318,80 @@ func initializeCheckpointManager(
 		return fmt.Errorf("failed to encode parameters for CheckpointManager.initialize. error: %w", err)
 	}
 
-	checkpointManagerAddress := ethgo.Address(manifest.RootchainConfig.CheckpointManagerAddress)
+	addr := ethgo.Address(manifest.RootchainConfig.CheckpointManagerAddress)
 	txn := &ethgo.Transaction{
-		To:    &checkpointManagerAddress,
+		To:    &addr,
 		Input: initCheckpointInput,
 	}
 
-	receipt, err := txRelayer.SendTransaction(txn, rootchainAdminKey)
+	return sendTransaction(txRelayer, txn, checkpointManagerName)
+}
+
+// initializeExitHelper invokes initialize function on "ExitHelper" smart contract
+func initializeExitHelper(txRelayer txrelayer.TxRelayer, rootchainConfig *polybft.RootchainConfig) error {
+	input, err := contractsapi.ExitHelper.Abi.GetMethod("initialize").
+		Encode([]interface{}{rootchainConfig.CheckpointManagerAddress})
+	if err != nil {
+		return fmt.Errorf("failed to encode parameters for ExitHelper.initialize. error: %w", err)
+	}
+
+	addr := ethgo.Address(rootchainConfig.ExitHelperAddress)
+	txn := &ethgo.Transaction{
+		To:    &addr,
+		Input: input,
+	}
+
+	return sendTransaction(txRelayer, txn, exitHelperName)
+}
+
+// initializeRootERC20Predicate invokes initialize function on "RootERC20Predicate" smart contract
+func initializeRootERC20Predicate(txRelayer txrelayer.TxRelayer, rootchainConfig *polybft.RootchainConfig) error {
+	input, err := contractsapi.RootERC20Predicate.Abi.GetMethod("initialize").
```

Comment:
Can we generate this initialize function in the `contractsapi` and use it here?

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1242/commits/06b992da38458ca3df75731995eacc950472894c